config/locales/translations/zendesk_apps_support.yml

```diff
@@ -130,16 +130,6 @@ parts:
       title: "App builder job: requirements file contains too many custom object trigger actions. The maximum number of actions allowed per trigger is 25. Leave requirements.json as is (do not translate)"
       value: "The requirements.json file contains too many custom object trigger actions. The current limit is %{max} actions per trigger. This app has %{count} actions for trigger %{trigger_title}."
       screenshot: "https://drive.google.com/file/d/18jXaM7_OnPqHAg6jAxJrTrgX1v_H156_/view?usp=sharing"
-  - translation:
-      key: "txt.apps.admin.error.app_build.excessive_cov2_dropdown_fields_per_object"
```

Comment:
This key and the following key have been already translated, please make them `obsolete` instead of removing. We recommend the date to be a bit further (2~3 months from now) in case.

https://zendesk.atlassian.net/wiki/spaces/globalization/pages/742392007/Delete+Translations